main_view.html

```diff
@@ -7,36 +7,33 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style_main_view.css">
+    
 </head>
 
 <body>
-    
+
+    <nav class="quick_acces">
+        <ul class="quick_access_ul">
+            <li class="close">X</li>
+            <li class="portfolio_go">Portfolio</li>
+            <li class="about_go">About</li>
+            <li class="contact_go">Contact</li>
+        </ul>
+        <img class="botton_line" src="images/Shape.svg" alt="">
+            </nav>
+    <header class="header_mobile">
+        <div class="menu">
+            <a href="#" class="logo"> Welcome </a>
+            <nav class="menu_img">
+                <img  class="menu_hamburguer" src="images/Enabledmenu_bar.svg" alt="">
+            </nav>
+        </div>
+    </header>    
```

Comment:
To match the Figma design:
- [ ] Kindly choose the right color and size of the close button.
- [ ] Please change the color of the `Portfolio` link to the right color as you did with the other options

| Your design | Figma design |
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/189947994-a0eabfa4-9f3d-4d81-927c-25d0b531bb74.png) | ![image](https://user-images.githubusercontent.com/24703171/189948190-0a23bce7-3201-413b-9213-db19dd16084e.png) |